truffle-config.js

```diff
@@ -68,6 +47,51 @@ const manualNetworks = {
   },
 }
 
+/**
+ * You shouldn't change anything below this line to preserve this Truffle Box's automatic handling of target realms,
+ * chains, networks, compiler versions, etc.
+ *
+ * If you want to configure anything related to networks, please customize the `manualNetworks` object above.
+ *
+ * Target realms (ethereum / boba / celo / conflux / etc.) are read from multiple possible sources, in this order:
+ *  1. `--network <realm>.<chain>` argument
+ *  2. `WITNET_EVM_REALM` environment variable
+ *  3. default (ethereum)
+ */
+const witnetSettings = require("./node_modules/witnet-ethereum-bridge/migrations/witnet.settings")
+let realm = process.env.WITNET_EVM_REALM ? process.env.WITNET_EVM_REALM.toLowerCase() : "default"
```

Comment:
`let realm = process.env.WITNET_EVM_REALM?.toLowerCase() ?? "default"`

Reply:
Requires node 14 though. Many people are still using node 10 or 12 :shrug:

Reply:
I did the same in https://github.com/witnet/witnet-price-feed-examples/pull/40